### PR TITLE
[MINOR] EDA-1319 official bot

### PR DESCRIPTION
**Pull request by ibrizuela-eb**
- An official bot now is created when a new user signs up.
- Move the class bot from development to auth_app.
- Migrations.

### CODE REVIEW

File: auth_app/models.py [hunk 1]
```diff
@@ -35,3 +39,24 @@ class User(AbstractBaseUser, PermissionsMixin):
 
     def __str__(self):
         return "@{}".format(self.username)
+
+
+@receiver(post_save, sender=User)
+def after_user_signed_up(sender, instance, **kwargs):
+    if Bot.objects.filter(user=instance,).count() == 0:
```
Comment (nicolasnunez-eb) [inline auth_app/models.py:46]: nit: you can use `if Bot.objects.filter(user=instance,).exists()`

File: auth_app/models.py [hunk 2]
```diff
@@ -35,3 +39,24 @@ class User(AbstractBaseUser, PermissionsMixin):
 
     def __str__(self):
         return "@{}".format(self.username)
+
+
+@receiver(post_save, sender=User)
```
Comment (nicolasnunez-eb) [inline auth_app/models.py:44]: Isn't there another part of the flow where you need to create a bot for a specific user?

File: auth_app/models.py [hunk 3]
```diff
@@ -35,3 +39,24 @@ class User(AbstractBaseUser, PermissionsMixin):
...
+
+
+class Bot(models.Model):
+    name = models.CharField(max_length=30)
+    token = models.CharField(max_length=200, default='')
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name='user',
+        null=True
```
Comment (nicolasnunez-eb) [inline auth_app/models.py:61]: Why a bot could have a user as null?

**Review by srobledo-eb — APPROVED** [review]

File: auth_app/models.py (hunk 2 above)
Comment (ibrizuela-eb) [reply auth_app/models.py:44]: In the future, it will be an option to add bots.

File: auth_app/models.py (hunk 1 above)
Comment (ibrizuela-eb) [reply auth_app/models.py:46]: I tried with `exists` but it doesn't create my bot.

File: auth_app/models.py (hunk 3 above)
Comment (ibrizuela-eb) [reply auth_app/models.py:61]: If I set null=False, now it asks me for a default value but I don't want a default value.

**Pull request merged**